lib/runtime/stream_table_ops.ts

```diff
@@ -167,6 +167,31 @@ class DelayedIterator<T> implements AsyncIterator<T> {
     }
 }
 
+export function tableJoin<T>(lhs : Stream<T>, rhs : Stream<T>) : AsyncIterator<ResultT<T>, void> {
```

Comment:
The type signature is not correct, it should be:
```
export function tableJoin<T1, T2>(lhs : Stream<T1>, rhs : Stream<T2>) : AsyncIterator<ResultT<T1 & T2>, void> {
```
given that it takes two streams and computes a stream with both properties.

Reply:
fixed in 2e7345d